Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -435,6 +436,28 @@ DasharoSystemFeaturesUiLibConstructor (
           );
       ASSERT_EFI_ERROR (Status);
     }
+
+  BufferSize = sizeof(mDasharoSystemFeaturesPSrivate.DasharoFeaturesData.BootManagerEnabled);
+  Status = gRT->GetVariable(
+      mBootManagerEnabledEfiVar,
+      &gDasharoSystemFeaturesGuid,
+      NULL,
+      &BufferSize,
+	    &mDasharoSystemFeaturesPrivate.DasharoFeaturesData.BootManagerEnabled
+      );
+  if (Status == EFI_NOT_FOUND) {
+    mDasharoSystemFeaturesPrivate.DasharoFeaturesData.mBootManagerEnabled = mBootManagerEnabledDefault;

```

Comment:
Does it even compile? I don't see mBootManagerEnabledDefault defined at all.

Reply:
Currently, I'm correcting the dependencies. This code compiles, but I have to make order in my local repos/working copies/submodules.

Reply:
Seems to be fixed in the latest rev.